src/Roew/ResultOrExceptionWrapper.php

```diff
@@ -43,7 +43,7 @@ abstract class ResultOrExceptionWrapper
      *
      * @throws \LogicException - When the operation succeeded
      */
-    abstract public function getException(): \Exception;
+    abstract public function getException(): void;
```

Comment:
This should be `\Exception`, if the wrapped operation failed, the `getException` method should return the thrown exception, if the operation succeeded, it should throw a logic exception.

see : https://github.com/azjezz/roew/blob/master/src/Roew/WrappedException.php#L60

Reply:
If this changes into `\Exception`, the `WrappedResult::getException` will have the error about return type cannot be compatible with abstract `ResultOrExceptionWrapper` class.

Reply:
both need to be `\Exception` :)